pkg/setup/config.go

```diff
@@ -64,5 +71,75 @@ func InitConfig(vConfig *config.VirtualClusterConfig) error {
 		}
 	}
 
+	// check if previously we were using eks or k8s as distro
+	if vConfig.Distro() != vclusterconfig.K8SDistro && vConfig.Distro() != vclusterconfig.EKSDistro {
```

Comment:
These checks won't work unfortunately as the /binaries folder is an empty dir which is wiped if the pod is recreated